packaging/suse/Portus.spec.in

```diff
@@ -74,12 +72,18 @@ install -d vendor/cache
 cp %{_libdir}/ruby/gems/%{rb_ver}/cache/*.gem vendor/cache
 export NOKOGIRI_USE_SYSTEM_LIBRARIES=1
 export PACKAGING=yes
+export IGNORE_ASSETS=no
```

Comment:
This is not required, given inside of the spec file we never defined this env variable.